nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/helpers/TestColdStorage.java

```diff
@@ -361,8 +383,7 @@ protected void addColdStorageContentBlobStatus(String docId, BlobStatus blobStat
         ManagedBlob coldContent = (ManagedBlob) session.getDocument(new IdRef(docId))
                                                        .getPropertyValue(
                                                                ColdStorageHelper.COLD_STORAGE_CONTENT_PROPERTY);
-
-        DummyBlobProvider blobProvider = (DummyBlobProvider) blobManager.getBlobProvider(coldContent.getProviderId());
-        blobProvider.addStatus(coldContent, blobStatus);
+        S3BlobProvider blobProvider = (S3BlobProvider) blobManager.getBlobProvider(coldContent.getProviderId());
+        //blobProvider.addStatus(coldContent, blobStatus);
     }
 }
```

Comment:
could you please add an extra line on this file on you cleanup/format commit. Thx in advance